src/BucketTesting/Validation/ValidationErrorLogger.php

```diff
@@ -22,6 +22,6 @@ public function getErrors(): array {
 	}
 
 	public function hasErrors(): bool {
-		return empty( $this->getErrors() );
+		return !empty( $this->getErrors() );
```

Comment:
Oops. Looks like this function was not tested in the last PR 😊

Reply:
Haha, the code was correct but the method name was wrong, I have fixed that in both places now.